app/lib/components/term.js

```diff
@@ -149,6 +160,10 @@ export default class Term extends Component {
     if (this.props.customCSS !== nextProps.customCSS) {
       this.term.prefs_.set('user-css', this.getStylesheet(nextProps.customCSS));
     }
+
+    if (this.devicePixelRatio !== window.devicePixelRatio) {
+      this.term.prefs_.set('font-smoothing', this.getFontSmoothing());
```

Comment:
It can, if say, you move from a non-retina screen to a retina screen. I have a retina macbook pro for work, and a secondary monitor that's non-retina. So if I move the window between screens, this would catch that and adjust as necessary 😄